### PR TITLE
zap log

### PR DESCRIPTION
**Pull request by veezhang**

### CODE REVIEW

File: pkg/controller/component/log.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import (
+	"github.com/go-logr/logr"
+
+	"github.com/vesoft-inc/nebula-operator/pkg/logging"
+)
+
+var _log = logging.Log.WithName("controller")
+
+func getLog() logr.Logger { return _log }
```
Comment (MegaByte875) [inline pkg/controller/component/log.go:27]: The function getLog()  has  some duplicates,  why not use 'log' directly?

File: pkg/controller/component/reclaimer/log.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+package reclaimer
+
+import (
+	"github.com/go-logr/logr"
+
+	"github.com/vesoft-inc/nebula-operator/pkg/logging"
+)
+
+var _log = logging.Log.WithName("controller")
```
Comment (MegaByte875) [inline pkg/controller/component/reclaimer/log.go:25]: reclaimer module append name "reclaimer" after "controller"?

File: pkg/kube/log.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+package kube
+
+import (
+	"github.com/go-logr/logr"
+
+	"github.com/vesoft-inc/nebula-operator/pkg/logging"
+)
+
+var _log = logging.Log.WithName("webhook")
```
Comment (MegaByte875) [inline pkg/kube/log.go:25]: WithName("kubeclient")?

File: pkg/logging/logging.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+	Opts    = runtimezap.Opts
+)
+
+var (
+	Log        = log.Log
+	LoggerFrom = ctrl.LoggerFrom
+	LoggerInto = ctrl.LoggerInto
+	SetLogger  = ctrl.SetLogger
+
+	New             = runtimezap.New
```
Comment (MegaByte875) [inline pkg/logging/logging.go:38]: what are these variables used for?

File: pkg/nebula/log.go [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+package nebula
+
+import (
+	"github.com/go-logr/logr"
+
+	"github.com/vesoft-inc/nebula-operator/pkg/logging"
+)
+
+var _log = logging.Log.WithName("webhook")
```
Comment (MegaByte875) [inline pkg/nebula/log.go:25]: WithName("nebulaclient")?

File: pkg/scheduler/extender/log.go [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+package extender
+
+import (
+	"github.com/go-logr/logr"
+
+	"github.com/vesoft-inc/nebula-operator/pkg/logging"
+)
+
+var _log = logging.Log.WithName("webhook")
```
Comment (MegaByte875) [inline pkg/scheduler/extender/log.go:25]: WithName("extender")?

File: pkg/scheduler/extender/predicates/log.go [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+package predicates
+
+import (
+	"github.com/go-logr/logr"
+
+	"github.com/vesoft-inc/nebula-operator/pkg/logging"
+)
+
+var _log = logging.Log.WithName("webhook")
```
Comment (MegaByte875) [inline pkg/scheduler/extender/predicates/log.go:25]: ditto

File: pkg/controller/component/log.go (hunk 1 above)
Comment (veezhang) [reply pkg/controller/component/log.go:27]: I was use `log` directly. 
But I found that the log is easy to be overwritten while using. Therefore, the global `log` variable has changed.

for example:

```golang
func example() {
    log = log.WithValues(...) // : missed
}
```

File: pkg/controller/component/reclaimer/log.go (hunk 2 above)
Comment (veezhang) [reply pkg/controller/component/reclaimer/log.go:25]: Which packages should be added to `WithName`?

1. Only top package
2. Only bottom package
3. Every package
4. Top and bottom packages

And I think 4 is better, it's be neither too many levels of packages nor too few.

**Review by MegaByte875 — APPROVED** [review]
LGTM

**Pull request merged**